app/settings.py

```diff
@@ -147,6 +147,10 @@
     ],
 }
 
+SIMPLE_JWT = {
+    "ROTATE_REFRESH_TOKENS": True,
+}
```

Comment:
Thank you!

Would it be possible to also add more configuration items like:
```
    "SIGNING_KEY": "some secret key, possibly autogenerated on deployment",
    "ACCESS_TOKEN_LIFETIME": datetime.timedelta(days=1),
    "REFRESH_TOKEN_LIFETIME": datetime.timedelta(days=7),
```

Reply:
Well, maybe nevermind the `SIGNING_KEY` as it defaults to the Django app `SECRET_KEY`.